src/ResultWriter/WaveFieldWriter.cpp

```diff
@@ -280,9 +280,11 @@ void seissol::writer::WaveFieldWriter::init(unsigned int numVars,
 	// Set up for low order output flags
 	m_lowOutputFlags = new bool[WaveFieldWriterExecutor::NUM_LOWVARIABLES];
 	m_numIntegratedVariables = seissol::SeisSol::main.postProcessor().getNumberOfVariables();
+
 	for (size_t i = 0; i < WaveFieldWriterExecutor::NUM_PLASTICITY_VARIABLES; i++) {
```

Comment:
NUM_LOWVARIABLES > NUM_PLASTICITY_VARIABLES, so the other entries are just left unitialised?

Reply:
I think they are initialized when calling seissol::SeisSol::main.postProcessor().getIntegrationMask

Reply:
ok